docs/Using-TensorFlow-Sharp-in-Unity.md

```diff
@@ -1,41 +1,53 @@
 # Using TensorFlowSharp in Unity _[Experimental]_
 
-Unity now offers the possibility to use pre-trained Tensorflow graphs inside of the game engine. This was made possible thanks to [the TensorFlowSharp project](https://github.com/migueldeicaza/TensorFlowSharp).
+ML-Agents allows you to use pre-trained [TensorFlow graphs](https://www.tensorflow.org/programmers_guide/graphs) inside your Unity games. This support is possible thanks to [the TensorFlowSharp project](https://github.com/migueldeicaza/TensorFlowSharp). The primary purpose for this support is to use the TensorFlow models produced by the ML-Agents own training programs, but a side benefit is that you can use any TensorFlow model.
 
 _Notice: This feature is still experimental. While it is possible to embed trained models into Unity games, Unity Technologies does not officially support this use-case for production games at this time. As such, no guarantees are provided regarding the quality of experience. If you encounter issues regarding battery life, or general performance (especially on mobile), please let us know._
 
 ## Supported devices :
 
  * Linux 64 bits
- * Mac OSX 64 bits
+ * Mac OS X 64 bits
  * Windows 64 bits
  * iOS (Requires additional steps)
  * Android
 
 ## Requirements
 
 * Unity 2017.1 or above
-* Unity Tensorflow Plugin ([Download here](https://s3.amazonaws.com/unity-agents/0.2/TFSharpPlugin.unitypackage))
+* Unity TensorFlow Plugin ([Download here](https://s3.amazonaws.com/unity-agents/0.2/TFSharpPlugin.unitypackage))
```

Comment:
Change to 0.3? @awjuliani  - presumably that's where the new package will be?

Reply:
Yep. Let's replace this tomorrow.

Reply:
Changed to 0.3 in link.